deliver/lib/deliver/runner.rb

```diff
@@ -45,8 +45,9 @@ def run
 
       UI.success("Finished the upload to iTunes Connect") unless options[:skip_binary_upload]
 
-      precheck_success = precheck_app
+      reject_version_if_possible if options[:reject_if_waiting_for_review]
 
+      precheck_success = precheck_app
       submit_for_review if options[:submit_for_review] && precheck_success
```

Comment:
When we call the `deliver` action without `reject_if_waiting_for_review`, and the app is in `can_reject` state, do you think we should show a message suggesting they can use this option to reject the version?

Reply:
I will change this and give it a shot ^